post_processing/plot/plot_helpers.jl

```diff
@@ -445,13 +410,17 @@ function generate_paperplots_held_suarez(fig_dir, nc_files; moist)
             global lon = nc["lon"][:]
             global z = nc["z"][:]
             global u = nc["u"][:]
+            global v = nc["v"][:]
```

Comment:
👀
Why are we using `global`s here?

Reply:
I think this is being done for the time mean plots? I was wondering the same myself; the intent of this PR is simply to replace `Makie` as our primary plot package.

Reply:
I think we just need `global` to be able to access these variables in the `else` block? I'm not 100% sure.

Reply:
Are `begin` blocks + `push!` better here than `global` + `cat` ?

Reply:
It's fine to clean up later, I was just surprised to see this

Reply:
Noted here: https://github.com/CliMA/ClimaAtmos.jl/issues/1816

Reply:
Thanks!